projects/packages/videopress/src/client/block-editor/blocks/video/edit.native.js

```diff
@@ -128,7 +132,7 @@ export default function VideoPressEdit( {
 			setAttributes( { guid: null } );
 			setFileToUpload( media );
 		},
-		[ setIsReplacingFile, setIsUploadingFile, setFileToUpload ]
+		[ attributes, setIsReplacingFile, setIsUploadingFile, setFileToUpload ]
```

Comment:
When replacing a video and stopping the upload, this missing dependency was making the block restore only part of the attributes and leaving it in a broken state.